packages/dropdown/src/js/index.ts

```diff
@@ -114,26 +110,21 @@ export const useDropdown = (
   props: UseDropdownProps,
   forwardedRef: Ref<HTMLButtonElement>
 ) => {
+  const id = uniqueId()
```

Comment:
please memoize. this will change on each render and could cause unnecessary dom updates